azurerm/internal/services/eventhub/eventhub_namespace_resource.go

```diff
@@ -332,6 +344,9 @@ func resourceArmEventHubNamespaceRead(d *schema.ResourceData, meta interface{})
 		d.Set("auto_inflate_enabled", props.IsAutoInflateEnabled)
 		d.Set("maximum_throughput_units", int(*props.MaximumThroughputUnits))
 		d.Set("zone_redundant", props.ZoneRedundant)
+		if dedicatedClusterID := props.ClusterArmID; dedicatedClusterID != nil {
+			d.Set("dedicated_cluster_id", dedicatedClusterID)
+		}
```

Comment:
this field always needs to be set - so can we remove the if statement here?

Reply:
Ah, just when I thought to be extra careful :)